cron_jobs/caravan_cron.php

```diff
@@ -135,6 +145,9 @@ function hourlyCaravan(System $system, $debug = true): void
             VALUES ('{$start_time}', '{$travel_time}', '{$region_id}', '{$village_id}', '{$caravan_type}', '{$resources}', '{$name}')";
     }
 
+    // all regions_locations should be emptied
+    $queries[] = "UPDATE `region_locations` SET `resource_count` = 0 WHERE `region_id` > 5";
```

Comment:
What is the significance of region_id > 5?

Reply:
Regions 1/2/3/4/5 are all home regions of Stone/Cloud/Leaf/Sand/Mist respectively
Since home regions do not have a castle, they don't use caravans. The resource move directly from the villages -> main village during each `region_cron` tick.
An improvement here would be to add a flag on the `regions` table, like `is_home_region`